### PR TITLE
Feature/booking

### PR DESCRIPTION
**Pull request by dev-catalyst**
### Ticket
closes #

### Checklist
- [ ] There are no warnings in the browser console
- [ ] I made sure the component is WCAG compliant using ANDI
- [ ] I made sure the UI follows the style guide in the wireframe (font, font-size, etc...)
- [ ] I made sure that the screen reader can read all the text
- [ ] I made sure I removed all the console.logs
- [ ] I made sure mobile portrait view looks good
- [ ] I made sure mobile landscape view looks good
- [ ] I added storybook for the component
	- [ ] Storybook not applicable
- [ ] I added unit test coverage
	- [ ] I added integration test coverage
	- [ ] Unit test coverage not applicable

### To Do
- [ ] 
- mentor dashboard
- mentee dashboard
- setting page
- booking dialogs
- responsiveness for all pages

### Additional Notes
screen shots for UI
profile_edit
setting-edit
confirm
confirmation
booknow
timeslot
dashboard
security
settings
schedule
dashboard-edit
schedule-mentee
settings
profile
security
schedule
booking
book
confirmation
dashboard
meetings
timeslot-add
timeslot-edit

### CODE REVIEW

File: src/components/Settings/TimeSlotModal/index.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    },
+    {
+      lable: 'F',
+      selected: false,
+    },
+    {
+      lable: 'S',
+      selected: false,
+    },
+  ]);
```
Comment (cfernando415) [inline src/components/Settings/TimeSlotModal/index.js:67]: can we create a variable outside the component for the initial state?

File: src/components/Settings/Profile.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        <h4>Personal meeting room</h4>
+        <div className={gooleMeet}>
+          <p className={'disabledFields'}>{googleMeet}</p>
+          <p
+            className={'disabledFields'}
+            style={{ background: 'white', cursor: 'pointer' }}
+          >
+            {'Copy Google Meet Link'}
+          </p>
+        </div>
```
Comment (cfernando415) [inline src/components/Settings/Profile.js:56]: i see some repeating patterns that could be written in another way, will have to look into this

File: src/components/Settings/Security.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                disabledInputs
+                  ? `${securityFields} ${disabledInputsBG}`
+                  : securityFields
+              }`,
+              disabled: disabledInputs,
+              error: !isEmpty(errors.retypePassword),
+              helperText:
+                errors.retypePassword && errors.retypePassword.message,
+            }}
+          />
```
Comment (cfernando415) [inline src/components/Settings/Security.js:126]: seeing some repeating patterns that could be written in another way, will have to look into this

**Review by cfernando415 — CHANGES_REQUESTED** [review]
pls see some comments, maybe we can take care of the easier ones, and we'll look into the repeating patterns later

File: src/components/Directory/BookDialog/SlotTable/index.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useStyles from './SlotTable.styles';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import { Button } from '@material-ui/core';
+// import Slider from 'react-slick';
+// import 'slick-carousel/slick/slick.css';
+// import 'slick-carousel/slick/slick-theme.css';
```
[the remark this answers is not in the document]
Comment (dev-catalyst) [reply src/components/Directory/BookDialog/SlotTable/index.js]: I have removed commented code

File: src/components/Directory/BookDialog/SlotTable/index.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    classes;
+  const days = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri'];
+
+  var meetTimeSlotsDataMobile = isMobile
+    ? meetTimeSlots.slice(0, 3)
+    : meetTimeSlots;
+
+  return (
+    <div className={root}>
+      {/* <Slider {...settings}> */}
```
[the remark this answers is not in the document]
Comment (dev-catalyst) [reply src/components/Directory/BookDialog/SlotTable/index.js]: I have removed commented code

File: src/components/Directory/BookDialog/SlotTable/index.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+                      disabled
+                    >
+                      unavailable
+                    </Button>
+                  ) : (
+                    slotItem.slots.map((slot, index) => {
+                      return (
+                        <Button
+                          variant="contained"
+                          // disabled={slot.status}
```
[the remark this answers is not in the document]
Comment (dev-catalyst) [reply src/components/Directory/BookDialog/SlotTable/index.js]: I have removed commented code

File: src/components/Directory/BookDialog/SlotTable/index.js [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+                          className={`${
+                            // slot.status ? bookedButton :
```
[the remark this answers is not in the document]
Comment (dev-catalyst) [reply src/components/Directory/BookDialog/SlotTable/index.js]: I have removed commented code

File: src/components/Directory/BookDialog/SlotTable/index.js [hunk 8]
```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useStyles from './SlotTable.styles';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import { Button } from '@material-ui/core';
+// import Slider from 'react-slick';
+// import 'slick-carousel/slick/slick.css';
+// import 'slick-carousel/slick/slick-theme.css';
+
+const SlotTable = ({ meetTimeSlots, handleMeetClose, handleConfirmOpen }) => {
```
[the remark this answers is not in the document]
Comment (dev-catalyst) [reply src/components/Directory/BookDialog/SlotTable/index.js:7]: added default value

File: src/components/Settings/Meetings.js [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+          <div>
+            <Button className={settingsButton}>Meet Now</Button>
+          </div>
+        </div>
+      ) : (
+        <div className={nomeetings}>
+          <img src={Calendar_icon} alt="calendar-icon" />
+          <p>No meetings yet!</p>
+          <p>Stay tuned.</p>
+        </div>
```
[the remark this answers is not in the document]
Comment (dev-catalyst) [reply src/components/Settings/Meetings.js]: done

File: src/components/Settings/Security.js [hunk 10]
```diff
@@ -0,0 +1,151 @@
...
+        </div>
+        {/* <Button
+          className={settingsButton}
+          onClick={() => setDisableInputs(false)}
+        >
+          Edit
+        </Button> */}
```
[the remark this answers is not in the document]
Comment (dev-catalyst) [reply src/components/Settings/Security.js:133]: I have removed commented code

File: src/components/Settings/TimeSlotModal/index.js (hunk 1 above)
Comment (dev-catalyst) [reply src/components/Settings/TimeSlotModal/index.js]: done

File: src/components/Settings/Profile.js (hunk 2 above)
Comment (dev-catalyst) [reply src/components/Settings/Profile.js:56]: I have removed the repetitive code by rendering html element through array

File: src/components/Settings/Security.js (hunk 3 above)
Comment (dev-catalyst) [reply src/components/Settings/Security.js:126]: I have removed the repetitive code by rendering html element through array

File: src/components/Directory/BookDialog/SlotTable/index.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Directory/BookDialog/SlotTable/index.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/150/commits/ef744c83f5fba486e0c1a5fcc2015df04202863d

File: src/components/Directory/BookDialog/SlotTable/index.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Directory/BookDialog/SlotTable/index.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/150/commits/ef744c83f5fba486e0c1a5fcc2015df04202863d

File: src/components/Directory/BookDialog/SlotTable/index.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Directory/BookDialog/SlotTable/index.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/150/commits/ef744c83f5fba486e0c1a5fcc2015df04202863d

File: src/components/Directory/BookDialog/SlotTable/index.js (hunk 7 above)
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Directory/BookDialog/SlotTable/index.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/150/commits/ef744c83f5fba486e0c1a5fcc2015df04202863d

File: src/components/Directory/BookDialog/SlotTable/index.js (hunk 8 above)
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Directory/BookDialog/SlotTable/index.js:7]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/150/commits/ef744c83f5fba486e0c1a5fcc2015df04202863d

[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Directory/BookDialog/SlotTable/index.js:7]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/150/commits/bc57a763ee612f222d9aca2d34ae878af1dcefb1

File: src/components/Settings/Meetings.js (hunk 9 above)
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Settings/Meetings.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/150/commits/1aa23f9ebcc562150dda72634554e89610df0e89

File: src/components/Settings/TimeSlotModal/index.js (hunk 1 above)
Comment (cfernando415) [reply src/components/Settings/TimeSlotModal/index.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/150/commits/8c42f0c7febd5499b0d628a6fc4ce0991d76a7dd

File: src/components/Settings/Security.js (hunk 3 above)
Comment (cfernando415) [reply src/components/Settings/Security.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/150/commits/6317b65d216f29280546a4824e5ffd1cddb15629

File: src/components/Settings/Profile.js (hunk 2 above)
Comment (cfernando415) [reply src/components/Settings/Profile.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/150/commits/6317b65d216f29280546a4824e5ffd1cddb15629

**Review by cfernando415 — APPROVED** [review]
:+1:

**Pull request merged**